receiver/dockerstatsreceiver/metadata.yaml

```diff
@@ -121,9 +121,15 @@ metrics:
       value_type: int
       monotonic: true
       aggregation: cumulative
+  container.cpu.utilization:
+    enabled: false
+    description: "Percent of CPU used by the container."
+    unit: "1"
+    gauge:
+      value_type: double
   container.cpu.percent:
     enabled: true
-    description: "Percent of CPU used by the container."
+    description: "Deprecated: use `container.cpu.utilization` metric instead. Percent of CPU used by the container."
     unit: "1"
```

Comment:
```suggestion
    unit: "1"
    warnings:
      if_enabled: This metric will be disabled in v0.82.0 and removed in v0.85.0.
```

Reply:
Addressed in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/22823/commits/551b12a69c22cd091e76c41270ee681d9b1cdfab